dotnet/src/Agents/OpenAI/Internal/AssistantThreadActions.cs

```diff
@@ -223,6 +218,12 @@ public static async IAsyncEnumerable<ChatMessageContent> GetMessagesAsync(Assist
                     // Block for function results
                     FunctionResultContent[] functionResults = await Task.WhenAll(functionResultTasks).ConfigureAwait(false);
 
+                    // Capture function-call for message processing
+                    foreach (FunctionResultContent functionCall in functionResults) // %%%
```

Comment:
What's the need for `// %%`?

Reply:
That an internal flag for myself that should've been removed.

Reply:
Updated